ipmininet/router/config/zebra.py

```diff
@@ -83,6 +86,59 @@ def has_started(self):
         # We override this such that we wait until we have the API socket
         return os.path.exists(self.zebra_socket)
 
+    @classmethod
+    def incr_last_routerid(cls):
```

Comment:
As the routerid should be shared accross daemons on the same router and must be different between router,
there are no benefits into having a per-daemon counter (so static should be fine).